test/SwapRouter.spec.ts

```diff
@@ -853,5 +853,64 @@ describe('SwapRouter', () => {
         })
       })
     })
+
+    describe('*WithFee', () => {
+      const feeRecipient = '0xfEE0000000000000000000000000000000000000'
+
+      it('#sweepTokenWithFee', async () => {
+        const amountOutMinimum = 100
+        const params = {
+          path: encodePath([tokens[0].address, tokens[1].address], [FeeAmount.MEDIUM]),
+          recipient: router.address,
+          deadline: 1,
+          amountIn: 102,
+          amountOutMinimum,
```

Comment:
our swap amount out minimum is different from the amount out after fee, it seems, which i guess is the responsibility fo the interface to handle